astropy/visualization/tests/test_norm.py

```diff
@@ -48,6 +53,11 @@ def test_scalar(self):
         assert_allclose(norm(6), 0.70710678)
         assert_allclose(norm(6), norm2(6))
 
+    def test_vmin_vmax_equal(self):
+        norm = ImageNormalize(vmin=2.0, vmax=2.0)
+        data = np.arange(10) - 5.0
+        assert_equal(norm(data), np.zeros(data.shape))
```

Comment:
This would do smart broadcasting.
```suggestion
        assert_array_equal(norm(data), 0)
```